lib/config.js

```diff
@@ -85,7 +85,8 @@ const defaultConfig = {
   withoutChromaSubsampling: false,
   compressionLevel: 6,
   format: null,
-  trim: false
+  trim: false,
+  fit: 'cover'
```

Comment:
The default option is `centre` https://sharp.pixelplumbing.com/api-resize#resize

Reply:
I think you're mistaking. According to the documentation the "fit" parameter default value is "cover". See here :
https://sharp.pixelplumbing.com/api-resize#resize
Centre is the "position" param default value.